src/runtime/basic-runtime.js

```diff
@@ -502,6 +502,10 @@ export class ConfiguredBasicRuntime {
           isClosable: !this.pageConfig().isLocked(),
         });
       };
+    } else if (options.autoPromptType == AutoPromptType.METER_LARGE) {
+      options.displayLargePromptFn = () => {
```

Comment:
So this would mean that we can either only show a subscription or meter prompt on a page, which may not be correct. The prompt itself depends on the result of the entitlements response so I feel like we should determine whether or not to prompt this [based on entitlements results ](https://github.com/subscriptions-project/swg-js/blob/98d20d7dc1d31762b6f7b8bc986d9d1b5b09ee6a/src/runtime/auto-prompt-manager.js#L124-L144)

Reply:
Reverted.